tools/kmod_scripts/sof_bootloop.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+echo "autospawn = no" > /etc/pulse/client.conf
+killall pulseaudio
```

Comment:
This has got to be one of the silliest fixes I've ever seen. You just erased the entire file with this, so all other settings are gone. Seriously?

Reply:
@plbossart updated